src/transformation/visitors/variable-declaration.ts

```diff
@@ -120,6 +124,93 @@ export function transformBindingPattern(
     return result;
 }
 
+export function transformBindingPattern(
```

Comment:
It transforms more than just binding pattern. Maybe `transform(Binding|Destructuring)VariableDeclaration`?